src/siembol/templates/siembol-deploy-job.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "siembol.deploy.alerting.fullname" . }}
+spec:
+  template:
+    spec:
+      containers:
+      - name: {{ template "siembol.deploy.alerting.name" . }}
+        image: "{{ .Values.deploy.image.repository }}:{{ .Values.deploy.image.tag }}"
+        command: ["sh", "-c", "storm -c nimbus.seeds='[\"{{ template "storm.nimbus.fullname" . }}\"]' jar alerting-storm-*.jar uk.co.gresearch.siembol.alerts.storm.AlertingStorm {{ tpl (.Files.Get "config/alerting.json") . | b64enc }}"]
```

Comment:
Calling `tpl` on the JSON config for the alerting Storm topology can sometimes fail with Helm (in my testing about 50% of the time, in a very random way it seems). Trying it again until it works is a (gross) workaround at the moment.
It looks like a bug in Helm. I'll try to hunt it down.

Reply:
I am now using a template defined in `_helpers.tpl` instead of reading a config file, which now works 100% of the time 😄